gcloud/bigquery/job.py

```diff
@@ -363,7 +483,7 @@ def allow_quoted_newlines(self):
 
     @property
     def create_disposition(self):
-        """Handling for missing destination table.
+        """Define how the back-end handles a missing destination table.
```

Comment:
4c4c3849999f560480e60be34d6e91d94eac94a0 adds a link to the URL of the relevant job configuration properties to the getter's docstring.